ocw/dataset_processor.py

```diff
@@ -68,8 +68,6 @@ def temporal_subset(month_start, month_end, target_dataset, average_each_year=Fa
 
     time_index = np.sort(time_index)
 
-    time_index = time_index[np.where((time_index >= time_index_first) & (time_index <= time_index_last))]
```

Comment:
Yes. The line is executed when `start_month < end_month` but has no effect on the `time_index` value.
This lines removes some `time_index` when `start_month > end_month`.
Please take a look at the test cases. Both test cases produce result as expected.